testkit/src/main/scala/org/bitcoins/testkit/BitcoinSTestAppConfig.scala

```diff
@@ -33,6 +33,25 @@ object BitcoinSTestAppConfig {
     BitcoinSAppConfig(tmpDir(), (overrideConf +: config): _*)
   }
 
+  def getSpvWithEmbeddedDbTestConfig(
```

Comment:
Maybe have something like this in a `TestDb.scala` 

```scala
  sealed trait TestDb {
    def createTestConfig(projectType: ProjectType): Config
  }
  
  object TestDb {
    case object Sqlite extends TestDb {
      override def createTestConfig(projectType: ProjectType): Config = ???
    }
    case class Postgres(pgUrl: String) extends TestDb {
      override def createTestConfig(projectType: ProjectType): Config = ???
    }
  }
```

and then maybe 

```scala
def configWithEmbeddedDb(db: TestDb, project: ProjectType): Config
``` ?